.opencode/agents/security-reviewer.md

```
@@ -8,47 +8,132 @@ permission:
 
-### Frontend (React/TypeScript) -- check when `web/src/` files changed
+```bash
+uv run python -m pytest tests/security -n 8
+uv run pre-commit run gitleaks --all-files
+uv run pre-commit run check-forbidden-literals --all-files
+npm --prefix web run lint
+```
+
+## Review Workflow
+
+### 1. Initial scan
```

markdownlint-cli2 (0.22.1)

[warning] Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Below

(MD022, blanks-around-headings)